### DISCUSSION TITLE
Add slight pause to audio when switching maps in browser

### DISCUSSION THREAD

**Discussion by lunatic-gmd** [ppy/osu#17582] [Ideas]
I don't know if anyone else has this issue, but switching between multiple maps in the in-game browser can get irritating and overwhelming because one song stops playing and immediately starts the next one without an audio break in between and that can give me some serious sensory overload if I do it 3 times or so.

(if I messed up the post at all, sorry, never used github before in my life)

**Comment by smoogipoo** [comment]
Is this issue for osu-lazer? There's already a slight fade when switching songs in song select.

**Reply by peppy** [reply]
The fade right now is a crossfade, while I think the proposal here is to actually perform a full fade-to-silence before fading in the new audio.

**Reply by lunatic-gmd** [reply]
yeah that's more what I meant. what ever osu stable does for that is really good imo

**Reply by peppy** [reply]
We went to great lengths to allow the lazer-style crossfade to work well, so that's disappointing to hear 😅.

I do think it can be tweaked to feel better than it currently does. Will be something I look at as part of a polish pass.

**Reply by lunatic-gmd** [reply]
Honestly, I don't wanna cause you a ton of trouble with that kinda thing, just more of a minor thing. If it's stressful to do anything with, I can play just fine without the change 😄
